src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -227,6 +229,77 @@ module.exports = class LevelPlane extends Array {
       this[myIndex].blockType = "redstoneWireCross";
     }
 
+    if (this[myIndex].isPowered) {
+      this[myIndex].blockType += "On";
+    }
+
     return this[myIndex].blockType;
   }
+
+  getRedstone() {
+    this.redstoneList = [];
+    this.redstoneListON = [];
+    for (let i = 0; i < this.length; ++i) {
+      if (this[i].blockType.substring(0, 12) === "redstoneWire") {
```

Comment:
Tiny style nit: `this[i].blockType.startsWith("redstoneWire")`

Reply:
Actually, why are we doing string compare on the block type?  Shouldn't these LevelBlock instances already have `this[i].isRedstone === true`?